homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -129,16 +129,19 @@ async def async_setup_entry(
 
     destination = config_entry.data[CONF_DESTINATION]
     origin = config_entry.data[CONF_ORIGIN]
-    name = f"{DEFAULT_NAME}: {origin} -> {destination}"
     region = config_entry.data[CONF_REGION]
 
     incl_filter = config_entry.options.get(CONF_INCL_FILTER)
     excl_filter = config_entry.options.get(CONF_EXCL_FILTER)
-    realtime = config_entry.options.get(CONF_REALTIME)
-    vehicle_type = config_entry.options.get(CONF_VEHICLE_TYPE)
-    avoid_toll_roads = config_entry.options.get(CONF_AVOID_TOLL_ROADS)
-    avoid_subscription_roads = config_entry.options.get(CONF_AVOID_SUBSCRIPTION_ROADS)
-    avoid_ferries = config_entry.options.get(CONF_AVOID_FERRIES)
+    realtime = config_entry.options.get(CONF_REALTIME, DEFAULT_REALTIME)
```

Comment:
Can we update the config entry options if the option is missing and use the config entry data items that were imported from configuration.yaml? If the option is missing in config entry data we use the default value.